ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6774,6 +6774,16 @@ VMADC:
   implements:
   - CONTROL
 
+MDSCV:
```

Comment:
Please re-name to be MDSDC (Motion-dependent supply damper control)

Reply:
Done